packages/configure-splash-screen/src/android/__tests__/Drawable.xml.test.ts

```diff
@@ -24,17 +42,7 @@ describe('Drawable.xml', () => {
     it('creates correct file', async () => {
       await configureDrawableXml(androidMainPath, ResizeMode.NATIVE);
       const actual = vol.readFileSync(filePath, 'utf-8');
-      const expected = `<?xml version="1.0" encoding="utf-8"?>
-<!--
-  This file was created by '@expo/configure-splash-screen' and some of it's content shouldn't be modified by hand
--->
-<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-  <item android:drawable="@color/splashscreen_background"/>
-  <item>
-    <bitmap android:gravity="center" android:src="@drawable/splashscreen_image"/>
-  </item>
-</layer-list>
-`;
+      const expected = generateDrawbleFileContent({ addBitmapItem: true });
```

Comment:
Hah! I found a typo! 😁
```suggestion
      const expected = generateDrawableFileContent({ addBitmapItem: true });
```